ds3_integration/smoke_test.go

```diff
@@ -342,11 +342,13 @@ func TestBulkGet(t *testing.T) {
 
     // Get all objects and verify content
     for _, obj := range availableChunks.Objects.Objects {
-        getObj, objErr := client.GetObject(models.NewGetObjectRequest(testBucket, *obj.Name))
-        ds3Testing.AssertNilError(t, objErr)
+        func() {
+            getObj, objErr := client.GetObject(models.NewGetObjectRequest(testBucket, *obj.Name))
+            ds3Testing.AssertNilError(t, objErr)
 
-        defer getObj.Content.Close()
-        testutils.VerifyBookContent(t, *obj.Name, getObj.Content)
+            defer getObj.Content.Close()
+            testutils.VerifyBookContent(t, *obj.Name, getObj.Content)
+        }()
```

Comment:
Was this `func` with the immediate invoke to force the defer to fire right away?

Reply:
Yes. If you look the defer is within a for loop, which will actually get kicked off at the end of the test. Its just bad form the way it was before. The IDE was giving me a warning. Figured I could clean up the test file real fast.